packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -241,7 +241,7 @@ export const ActionDetailsModalListTable = ({
         columns={commentsColumn}
         content={actionDetailsData.comments
           .concat(actionDetailsData.statusReason?.text || [])
-          .map((text: string) => ({ comment: text }))}
+          .map((text: any) => ({ comment: text.comment }))}
```

Comment:
The absolute minimum would probably be 

```suggestion
          .map((text: { comment: string }) => ({ comment: text.comment }))}
```

`any` indicates that the value can be literally anything, including number, boolean... that do not have a `comment` property